pkg/transform/kubernetes/kubernetes.go

```diff
@@ -106,6 +106,12 @@ func (t *Transformer) CreateDeployments(o *object.Service) ([]runtime.Object, er
 		},
 	}
 
+	// Set replica value only if it is given
+	if o.Replicas > 0 {
+		replicas := o.Replicas
+		d.Spec.Replicas = &replicas
```

Comment:
@containscafeine i thought it will have to save entire object `o` to just assign value, until `d` is alive. But if I can copy it in some variable then it will be less wastage of resources?